src/components/Questions/Footer.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div
+    class="flex w-full bg-white p-6 bp-500:p-8 md:p-10 lg:p-12 justify-between"
+  >
+    <div class="place-self-start flex h-full">
+      <!-- back button -->
+      <icon-button
+        :iconConfig="previousQuestionButtonIconConfig"
+        buttonClass="bg-yellow-500 hover:bg-yellow-600 ring-yellow-500 p-2 px-6 bp-500:p-4 bp-500:px-8 rounded-2xl shadow-xl"
+        ariaLabel="Previous Question"
+        v-if="isPreviousButtonShown"
+        @click="gotToPreviousQuestion"
```

Comment:
Why only previous question and not next question?

Reply:
Because going to the next question is handled in the submit button itself. you can refer to the design here: https://www.figma.com/file/4IwP0VyYqV3ZoUANndk9Fz/Quizzes-Mobile-Version?node-id=0%3A1